test/test_sparse.py

```diff
@@ -1419,6 +1424,24 @@ def test_sparse_add_out_bfloat16(self, device, dtype, coalesced):
         res_bf16 = res_bf16.float()  # to compare with reference
         self.assertEqual(res_fp32, res_bf16, atol=1e-2, rtol=0)
 
+    @onlyCPU
+    @coalescedonoff
+    @dtypes(torch.double, torch.cdouble)
+    def test_sparse_add_out_bfloat16_cpu(self, device, dtype, coalesced):
```

Comment:
Why is this additional test needed as opposed to extending an existing test for addition?